bin/growpart

```diff
@@ -447,7 +447,8 @@ resize_sfdisk() {
 	debug 1 "max_end=${max_end} tot=${sector_num} pt_end=${pt_end}" \
 		"pt_start=${pt_start} pt_size=${pt_size}"
 	[ $((${pt_end})) -eq ${max_end} ] && {
-		nochange "partition ${PART} is size ${pt_size}. it cannot be grown"
```

Comment:
Needs a ‘local’ declaration for part_size, right? And might as well add that to the debug statement

Reply:
Thanks, added 'local' declaration and its value to the debug statement.

Reply:
What I don't like here is that we have `pt_size` in sectors and `part_size` in bytes, which I find confusing. Maybe the new variable should be called `pt_size_bytes` (should suggest: "same as pt_size but in bytes").

However my real suggestion here is to keep using `pt_size` as is and just add the word "sectors" to the error message:
```
nochange "partition ${PART} is size ${pt_size} sectors. it cannot be grown"
```
or:
```
nochange "partition ${PART} is size ${pt_size} ${sector_size}-byte sectors. it cannot be grown"
```
After all when dealing with partitions we use sectors: if I do `fdisk` `p` to check what's going on with the partitions what I get are sector numbers, not bytes.

Reply:
Thanks, @paride  showing pt_size as num of sectors mostly like debug information which are listed in '-v'.
For end user, the size in bytes or other unit might be more straightforward in understanding.
@smoser what's your thoughts?

Reply:
I don't have strong feelings.  I do think that sectors are not meaningful to a human. I'm fine with either of the suggestions here:
 a.) report output in bytes 
 b.) report output in sectors including the sector size

but i'm also fine with:

 c.)  partition ${PART} is size 732422144 bytes (1430512 sectors of 512). It cannot be grown.

So... I'll just take myself out of the conversation.

Reply:
`growpart -v` does probably give enough info after all, so I'm also fine with either a), b) or c). On the variable name: `disk_size` is in bytes, so naming the variable `pt_size_bytes` doesn't save us from having a mix.

Bottom line: @liangxiao1 up to you what to amend (if anything). Please ping when this is ready for another review.

Reply:
LGTM, thanks!